content/christine_belzie/create-a-url-using-slugs.md

```diff
@@ -0,0 +1,77 @@
+---
+Title: "Create a URL Using Slugs"
+Description: "Step-by-Step guide on creating a slug with URL ."
+DatePublished: "2022-01-14"
+Categories:
+  - "web-development"
+Tags:
+  - "World Wide Web"
+  - "Link"
+CatalogContent:
+  - "introduction-to-the-web"
+  - "paths/web-development"
+---
+
+[https://downshiftology.com/recipes/gluten-free-chocolate-chip-cookies/]: https://downshiftology.com/recipes/gluten-free-chocolate-chip-cookies/
+[URL]: https://www.codecademy.com/resources/docs/general/url
+[slugify.online]: https://slugify.online/
+[John Mueller from Google(start at 19:00)]: https://www.youtube.com/live/4FkSZIW6d48?feature=share
+
+[Step 1 of URL slug]: https://github.com/Codecademy/ugc/main/content/christine_belzie/step1-of-slug.png
+[Step 2 of URL slug]: https://github.com/Codecademy/ugc/main/content/christine_belzie/step2-of-slug.png
+[Step 3 of URL slug]: https://github.com/Codecademy/ugc/main/content/christine_belzie/step3-of-slug.png 
+
+## Introduction 
+
+Picture this. You're scrolling through the internet, find a cookie recipe, and click on the "Share" button to send this link to your friend:
+[https://downshiftology.com/recipes/gluten-free-chocolate-chip-cookies/]
+
+You see `gluten-free-chocolate-chip-cookies` after the backslash (`/`). This is called a **slug**. Slugs are a vital part of a [URL] that helps identify a particular page on a website in a user-friendly manner. Essentially, slugs identify the content found on the page. In this article, you will learn the importance of slugs in SEO strategy, how to create them, and best practices.   
+
+## Why Are Slugs Important to SEO Strategy?
+
+Even though slugs won't have a direct impact on how your content ranks on the web, they do influence a visitor's decision on whether to view your content or not. Here's how:
+
+1. **Increases Search Ranking:** Cannibalization is a common issue for content creators. It's when most of the blog posts, articles, and other pieces of content on a website share the same keyword. This makes it difficult for people to find them on your site. Slugs solve this issue by putting a unique name like a title or heading, thus, making it easier for people to find a person's work. Think of this as the names of the folders on your Google Drive or OneDrive account. 
+
+2. **Improves Shareability:** Adding slugs to your content's URLs makes them shorter, thus, making it easier for your viewers to copy, paste, and send them to other people. 
+
+Now, let's create a slug! 
+
+## How To Create a Slug 
+
+Consider using the following methods to create a slug:
+
+### Method 1: Manually 
```

Comment:
This offers a good overview! I believe it would be even more helpful if it included detailed instructions. Specifically, I think this section could serve as the bulk of the tutorial.

Additionally, it would be great if you could incorporate the later section on best practices into this section. It seems that the online tool you mentioned can automatically handle this task, so incorporating best practices with this manual method may be an easier read

Reply:
Sure I can try that.